src/modules/Course/components/CardClassroom/index.tsx

```diff
@@ -1,27 +1,25 @@
 import { Avatar, Button, Card } from 'flowbite-react';
 import React from 'react';
-import { HiCalendar, HiClock, HiInbox, HiUser } from 'react-icons/hi';
+import { IconType } from 'react-icons/lib';
 
 interface Classroom {
   title: string;
+  subtitle?: string;
 }
 
 interface Info {
-  data: string;
-  time: string;
-  educator: string;
-  modality: string;
+  icon: IconType;
+  name: string;
 }
 
 interface Students {
-  avatar?: string;
-  title: string;
+  avatar: string;
   text?: string;
 }
 
 interface CardClassroomProps {
   classroom: Classroom;
-  info: Info;
+  info: Info[];
   students: Students;
```

Comment:
Isso aqui não deveria ser uma array?

Reply:
@rluders oque deveria ser array? Students and classroom?